src/Doctrine/ORM/GiftCardRepository.php

```diff
@@ -69,4 +69,14 @@ public function createAccountListQueryBuilder(CustomerInterface $customer): Quer
 
         return $qb;
     }
+
+    public function findCreatedBefore(\DateTimeInterface $date): array
+    {
+        $qb = $this->createQueryBuilder('gc');
+
+        $qb->andWhere('gc.createdAt < :date');
+        $qb->setParameter('date', $date->format('Y-m-d H:i:s'));
```

Comment:
```suggestion
        $qb->setParameter('date', $date);
```

Isn't this possible?